<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RKK" ignored="1" name="TestSmoke" tests="2" failures="1" timestamp="2022-12-31T13:04:14 IST" time="19.439" errors="0">
  <testcase name="VerifyLogin" time="1.14" classname="com.SauceLabTestClasses.LoginPageTest"/>
  <testcase name="VerifyLoginPageURL" time="0.029" classname="com.SauceLabTestClasses.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com] but found [https://www.saucedemo.com/]
at com.SauceLabTestClasses.LoginPageTest.VerifyLoginPageURL(LoginPageTest.java:35)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginPageURL -->
  <testcase name="VerifyLoginPageTitle" time="0.0" classname="com.SauceLabTestClasses.LoginPageTest">
    <ignored/>
  </testcase> <!-- VerifyLoginPageTitle -->
</testsuite> <!-- TestSmoke -->
